configlet-ci/fetch-configlet

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
+set -euo pipefail
 
-# https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
-get_latest_release() {
-  curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
-    grep '"tag_name":' |                                            # Get tag line
-    sed -E 's/.*"([^"]+)".*/\1/'                                    # Pluck JSON value
+get_download_url() {
+  # Returns the download URL of the latest configlet Linux release from the GitHub API
+  local api_url='https://api.github.com/repos/exercism/configlet/releases/latest'
+  local asset_name='configlet-linux-64bit.tgz'
+  curl -H "Accept: application/vnd.github.v3+json" -sSf "${api_url}" |
```

Comment:
In a script, there's no reason not to use the long options
```bash
  curl --header "Accept: application/vnd.github.v3+json" --silent --show-error --fail "${api_url}" |
```
or
```bash
  curlopts=(
    --header "Accept: application/vnd.github.v3+json"
    --silent
    --show-error
    --fail
  )
  curl "${curlopts[@]}" "${api_url}" |
```

Reply:
Good idea, thanks. We can also use this to specify the common curl options only once.

I added a new commit.